DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/Resources.resx

```diff
@@ -0,0 +1,281 @@
+              <xsd:attribute name="mimetype" type="xsd:string" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="assembly">
+            <xsd:complexType>
+              <xsd:attribute name="alias" type="xsd:string" />
+              <xsd:attribute name="name" type="xsd:string" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ActivityArgumentAnalyzerDescription" xml:space="preserve">
+    <value>Activity Function call specifies a parameter that dosn't match the function definition parameter</value>
+  </data>
+  <data name="ActivityArgumentAnalyzerMessageFormat" xml:space="preserve">
+    <value>Activity Function named '{0}' takes a '{1}' but was given a '{2}'</value>
+  </data>
+  <data name="ActivityArgumentAnalyzerTitle" xml:space="preserve">
+    <value>Activity Function call is using the wrong argument type</value>
+  </data>
+  <data name="ActivityNameAnalyzerCloseMessageFormat" xml:space="preserve">
+    <value>Activity function named '{0}' does not exist. Did you mean '{1}'?</value>
+  </data>
+  <data name="ActivityNameAnalyzerDescription" xml:space="preserve">
+    <value>Activity Function call references unknown Activity Function</value>
```

Comment:
For the message format:
Yes because {0} is the location of the actual diagnostic. {1} is an activity function that does exist with the most similar name. This is really useful for customers who might be barely off (EX: "ProcessString" vs "ProcessStrings"). This helps them fix their code easily without having to search through all of their function definitions to see which function they might have intended to call.

For the description:
This is a general description without a formatted string